src/parse/preprocess.ts

```diff
@@ -80,10 +50,59 @@ export function preprocessTemplateRange(
 
   // We need to replace forward slash with _something else_, because
   // forward slash breaks the parsed templates.
-  const content = template.contents.replaceAll('/', PLACEHOLDER);
-  const tplLength = template.range.end - template.range.start;
```

Comment:
None of this code is needed as content-v4 includes character ranges

Reply:
Similarly to above, let's look into removing more code in a separate task?